src/emulator/extensionsEmulator.ts

```diff
@@ -169,4 +184,39 @@ export class ExtensionsEmulator {
       STORAGE_BUCKET: `${projectId}.appspot.com`,
     };
   }
+
+  private async checkAndWarnAPIs(instances: planner.InstanceSpec[]): Promise<void> {
+    const apisToWarn = await getUnemulatedAPIs(this.args.projectId, instances);
+    if (apisToWarn.length) {
+      const table = new Table({
+        head: [
+          "API Name",
+          "Instances using this API",
+          `Enabled on ${this.args.projectId}`,
+          `Enable this API`,
```

Comment:
generally lgtm, something to think about with UX, there could be benefit to put this outside of the table - it makes it clear it is an optional step, maybe the user doesn't want to enable this API until they do more testing, see the logs, or they simply don't want to incur any cost etc